haskell/upload-hackage-docs.sh

```
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+# Authors: @maksbotan, @gromak
+
+# Use this script if you want to manually upload documentation to Hackage.
+# Sometimes Hackage fails to build your package and you don't want to fix
+# it but want to make your documentation available.
+# You need to specify:
+# • ${PACKAGE_NAME} is a name of a package you are uploading docs for
```

BTW, maybe fail if not specified?

---

I have `set -u` in the beginning.